replica/table.cc

```diff
@@ -1150,6 +1150,7 @@ future<bool> table::perform_offstrategy_compaction() {
 }
 
```

Comment:
please replace 11901 by https://github.com/scylladb/scylladb/issues/1239, and close the former.

Reply:
sure

Reply:
Fixed in v2